README.md

```diff
@@ -131,6 +131,17 @@ some UI utilities that make easy to use them as a development tool.
 
 🔗 [**Read More: Use your testing tool as your primary development tool**](/sections/generic-best-practices/use-your-testing-tool-as-your-primary-development-tool.md)
 
+<br/>
+
+## ![✔] 2.6 Keep abstraction low to ease debugging the tests
+
+**TL;DR:** Tests should be written with readability and debuggability in mind. Abstraction may be good in some instances, but it always incurs a cost in debuggability and therefore sometimes may not be worth it. This is especially important when speaking about UI tests since the complex stack get harder to understand the real source of failure. Reducing abstraction for the sake of easier debugging is key for future proofing the test code.
```

Comment:
Can we tweak this slightly?

*This is especially important when speaking about UI tests since the complex stack get harder to understand the real source of failure*

This is especially important for UI tests; consequent of the complex stack, it gets harder to understand the real source of failurer.

Reply:
Done 😊